internal/controller/pullrequest_controller.go

```diff
@@ -60,119 +60,79 @@ func (r *PullRequestReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	logger := log.FromContext(ctx)
 
 	var pr promoterv1alpha1.PullRequest
-	err := r.Get(ctx, req.NamespacedName, &pr, &client.GetOptions{})
-	if err != nil {
+	if err := r.Get(ctx, req.NamespacedName, &pr); err != nil {
```

Comment:
Inline checks break my brain a little, especially if it's more than just the nil check. Feel strongly about inlining it?

Reply:
I like them but I am also fairly impartial